crates/glue/src/load.rs

```diff
@@ -242,6 +215,69 @@ pub fn generate(
     }
 }
 
+fn call_roc_make_glue(
+    lib: &Library,
+    backend: CodeGenBackend,
+    roc_types: roc_std::RocList<roc_type::Types>,
+) -> roc_std::RocList<roc_type::File> {
+    let roc_call_result = match backend {
+        CodeGenBackend::Assembly(_) => {
+            type MakeGlueReturnType = RocCallResult<
+                roc_std::RocResult<roc_std::RocList<roc_type::File>, roc_std::RocStr>,
+            >;
+            type MakeGlue =
+                unsafe extern "C" fn(roc_std::RocList<roc_type::Types>) -> MakeGlueReturnType;
+
+            let name_of_main = "test_main";
```

Comment:
So we don't have a proper main/_exposed yet, but this is just a workaround?

Reply:
we do, this is proper in that it uses the same mechanism as the gen tests. We need the setjmp/longjmp insertion here. 

but yeah this needs to be made more consistent at some point